### PR TITLE
Set default scope-prefix to none

### LINKED ISSUE #46: Default to no prefix and/or allow eliminating prefix
It doesn't seem like there's a way to eliminate the prefix within —scope-prefix.

This is a surprising behavior that doesn't seem to have a purpose I can imagine, when you can write those prefixes in your own code.

I would argue that the default should be no prefix, and then that config option will still be available to automatically add prefixes.

### PR DESCRIPTION
**Pull request by mattiarossi**
Fix #46

* set default prefix to none,
* updated readme
* updated tests
* added example for common types included from a different folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Generate PLV8 functions for an input typescript file
...
 | --output-folder         | String                                | Specify an output folder                                                                                                                                                                                                                                                                | `plv8ify-dist` |
-| --scope-prefix          | String                                | Specify a scope prefix, by default `plv8ify`, adds `plv8ify_` as prefix for exported typescript functions                                                                                                                                                                               | `plv8ify`      |
+| --scope-prefix          | String                                | Specify a scope prefix, by default `empty`, adds provided string as prefix for exported typescript functions                                                                                                                                                                            | `plv8ify`      |
```
Comment (divyenduz) [inline README.md:186]:
```suggestion
| --scope-prefix          | String                                | Specify a scope prefix, by default '' (empty string), adds provided string as prefix for exported typescript functions                                                                                                                                                                            | `plv8ify`      |
```

**Review by divyenduz — APPROVED** [review]
Implementation looks good, thanks. 

1. Added a suggestion for readme
2. You might want to re-generate all examples, the filename/function name for them should change correct? Since we changed the default scope. I think you can use `yarn examples` to re-generate all examples! (and add any missing examples to that command)

Thank you 🙌🏼

**Comment by mattiarossi** [comment]
Recreated / Renamed examples and updated README and package.json

**Review by divyenduz — APPROVED** [review]
Great work! Thank you so much

**Comment by divyenduz** [comment]
This is live in [0.0.56](https://github.com/divyenduz/plv8ify/commit/20edcd11c9b445582ca69de2ca79ad53233c80c3), cc @braco

**Pull request merged**